index.html

```diff
@@ -1608,7 +1608,31 @@ <h3>Defining 'string'</h3>
 	</div>
 </section>
 
+<section id="char_whitespace" class="subtopic">
+	<h3>Whitespace characters</h3>
+	
+	<div class="xref"><span class="seealso">See also</span>
+	<p>[[[#markup_identifiers]]].</p>
+	</div>
 
+	<p>
+		Whitespace characters are characters that represent horizontal or vertical space in typography. Different whitespace characters may have different lengths.
```

Comment:
> Different whitespace characters may have different lengths.

This is a bit unclear--partly because "length" suggests different numbers of code points or code units, whereas I think you mean "amounts of visible whitespace on the page"? The trick here is that there are some whitespace characters that result in _no_ visible white space on the page. Perhaps:

> Whitespace characters can have different visual effects: some whitespace characters have no visible effect, while others represent larger, smaller, or variable amounts of space on the page.

Reply:
Fixed. Thanks.